solutions/toc.yml

```diff
@@ -7,20 +7,44 @@ items:
     - name: dataminer.MediaOps
       topicUid: MediaOps
       items:
-      - name: Installing dataminer.MediaOps
-        topicUid: Installing_MediaOps
-      - name: MediaOps applications
-        topicUid: MediaOps_apps
+      - name: Resource Studio
+        topicUid: MO_Resource_Studio
+        items: 
+        - name: Resource Icons
+          topicUid: RS_Resource_Icons
+        - name: Linking resource pools
+          topicUid: RS_Linking_Pools
+      - name: Scheduling
+        topicUid: MO_Scheduling
         items:
-        - name: People & Organizations
-          topicUid: People_Organizations
-          items:
-          - name: Managing contacts
-            topicUid: PO_Managing_Contacts
-          - name: Managing teams
-            topicUid: PO_Managing_Teams
-          - name: Managing organizations
-            topicUid: PO_Managing_Organizations
+          - name: Creating a Job
+            topicUid: SCH_Create_Job
+          - name: Editing a Job
+            topicUid: SCH_Edit_Job
+          - name: Job states
+            topicUid: MO_S_Job_States
+          - name: Job Locking
+            topicUid: MO_S_Job_Locking
+          - name: Validation of (Upcoming) Jobs
+            topicUid: Overview_MediaOps_Validation
+      - name: Workflow Designer
+        topicUid: MO_Workflow_Designer
+        items:
+          - name: Creating Workflows
+            topicUid: WFD_Creating_Workflows
+      - name: People & Organizations
+        topicUid: People_Organizations
+        items:
+        - name: Managing contacts
+          topicUid: PO_Managing_Contacts
+        - name: Managing teams
+          topicUid: PO_Managing_Teams
+        - name: Managing organizations
+          topicUid: PO_Managing_Organizations
+      - name: Tutorials
```

Comment:
@EnverPO  Not all tutorials are added here. You can navigate from the tutorials page to them, but you lose the surveyor.

Reply:
@EnverPO  just checked this one and is not yet solved